src/Draco.Compiler/Internal/Binding/LocalBinder.cs

```diff
@@ -169,7 +169,9 @@ private void Build()
     {
         FunctionDeclarationSyntax function => new SourceFunctionSymbol(this.ContainingSymbol, function),
         ParameterSyntax parameter => new SourceParameterSymbol((FunctionSymbol)this.ContainingSymbol, parameter),
-        VariableDeclarationSyntax variable => new SourceLocalSymbol(this.ContainingSymbol, new TypeVariable(localCount), variable),
+        VariableDeclarationSyntax variable => variable.FieldKeyword is null ?
+            new SourceLocalSymbol(this.ContainingSymbol, new TypeVariable(localCount), variable) :
+            new SourceFieldSymbol((TypeSymbol)this.ContainingSymbol, variable),
```

Comment:
Yeah let's revert as locally a field keyword will be illegal. I'll try to wrap up the properties PR which handles this case

Reply:
Revert to what ?

Reply:
```cs
VariableDeclarationSyntax variable => new SourceLocalSymbol(this.ContainingSymbol, new TypeVariable(localCount), variable),
```
Again, in local context there is no concept of fields